Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1380,6 +1380,12 @@ private string GetCommandLineArguments(string outputPath, StreamState state, boo
             var outputExtension = GetSegmentFileExtension(state.Request.SegmentContainer);
             var outputTsArg = outputPrefix + "%d" + outputExtension;
 
+ var ReplayGain = '';
+            if (ifUserConfiguration.ReplayGain = 1)
+            {
+                ReplayGain = string.format(" -af volume=replaygain=track");
```

Comment:
var ReplayGain is null unless the config is true, then it is assigned the value.